cpp/src/arrow/buffer.cc

```diff
@@ -47,6 +48,15 @@ Status Buffer::Copy(const int64_t start, const int64_t nbytes,
   return Copy(start, nbytes, default_memory_pool(), out);
 }
 
+void Buffer::print() {
+  const uint8_t* _data = is_mutable() ? mutable_data() : data();
+
+  for (int64_t i = 0; i < size(); ++i) {
+    std::cout << std::hex << static_cast<unsigned int>(_data[i]);
+  }
+  std::cout << std::endl;
```

Comment:
How about returning `std::string` instead of outputting to `std::count`?
`std::cout` may not be a suitable output sometimes. For example, `std:cerr` or outputting to file will be better in a command in pipeline "curl ... | arrow_command | another_command` case.

Reply:
You are rigth. It's done.